### DISCUSSION TITLE
请求帮助

### DISCUSSION THREAD

**Discussion by Bahajsjsh** [share121/inter-knot#3289] [其他]
![Screenshot_20240820_161621](https://github.com/user-attachments/assets/98da32e6-67b3-48d7-ab7d-3a501db4e02e)
我是小白不知道怎么弄求告知

**Comment by shan-dgha** [comment]
你的梯子不行，就是网不稳定导致的

**Comment by getmotos** [comment]
这个我也遇到了，我是按照up主说的把页面缓存清了之后就好了但是我也不知道具体是不是这个原因，  
[这个](https://www.bilibili.com/video/BV1MPWAe3E23/?spm_id_from=333.337.search-card.all.click&vd_source=0854e1fd24793ad7f385851db2fa22d9)  
<img src="https://bbs-static.miyoushe.com/static/2023/11/16/810dd11a4d16759f16ca078596aab6b1_422007669816866994.png" alt="哲-让我想想" width="75" >

**Reply by LuAn-afl** [reply]
就是缓存问题清理就好了我一直都是这样的

**Comment by Bahajsjsh** [comment]
OK现在不需要了已经搞好了
